providers/tpm2/tox.ini

```diff
@@ -4,9 +4,12 @@ skip_missing_interpreters = true
 skipsdist=True
 
 [testenv]
+whitelist_externals = rm
```

Comment:
According to [the documentation](https://tox.wiki/en/latest/changelog.html#v3-18-0-2020-07-23), this parameter has been deprecated:

> Add `allowlist_externals` alias to `whitelist_externals` (`whitelist_externals` is now deprecated). - by [@dajose](https://github.com/dajose) [#1491](https://github.com/tox-dev/tox/issues/1491)

Might be better to stick to the new alias to avoid problems later :)

Reply:
thanks for the review, this is fixed.